tests/system/test_root_metronome.py

```diff
@@ -239,6 +239,28 @@ def test_docker_job():
         assert len(client.get_runs(job_id)) == 1
 
 
+def test_metronome_shutdown_with_no_extra_tasks():
+    """ Test for METRONOME-100 regression
+        When Metronome is restarted it incorrectly started another task for already running job run task.
+    """
+    client = metronome.create_client()
+    job_id = "metronome-shutdown-{}".format(uuid.uuid4().hex)
+    with job(job_no_schedule(job_id)):
+        # run a job before we shutdown Metronome
+        client.run_job(job_id)
+        time.sleep(2)
```

Comment:
I know that this is potentially flaky but this is a common pattern in these tests right now. I would rather change that in a separate PR addressing this https://jira.mesosphere.com/browse/METRONOME-212

Reply:
Those sleeps are meh. We tried to remove them in marathon tests - no need to start with this here. Parsing the result of `run_job(job_id)` should give you the jobId (e.g. `"id": "20160715130259A34HX"`). Querying that from `v1/jobs& activeRuns=true&history=true&scheduled=true` should show when the run ends. So `deployment_wait`.